ref/general/data-persistence-jpa.adoc

```diff
@@ -19,7 +19,7 @@ Persistent data is data that outlives the application process that created it. A
 
 == What is JPA ?
 
-Enterprise applications require efficient access to relational databases. For example, an event calendar application needs to create, retrieve, update, and delete the events that users add to their calendar. The application itself can't efficiently store this data and must be able to quickly retrieve it from a database, complete the requested function, and if necessary, return it to persistent storage.
+Enterprise applications require efficient access to relational databases. For example, an event calendar application needs to create, retrieve, update, and delete the events that users add to their calendar. The application itself can't efficiently store this data and must be able to quickly retrieve it from a database, complete the requested function, and if necessary, return it to persistent storage. In the past, to configuring these tasks required developers to manage complex SQL operations and differences among vendor databases. JPA provides an extra level of abstraction between the application and the database, enabling developers to manage data persistence in a standardized way, by using annotations or XML.
```

Comment:
Update:
"In the past, configuring these..."

It looks like there's an extra "to" in there.